src/components/Answer/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+import * as classes from './style.module.css';
+
+type Props = {
+  label: string;
+  isCorrect: boolean;
```

Comment:
Eu trocaria essa prop e o estado interno por uma simples prop `state` com os 3 possíveis valores (não respondido, certo, errado). Do jeito que está os componentes pai e filho compartilham a responsabilidade desse estado e não fica claro quem cuida do que. Do jeito que estou propondo o componente da opção é "burro" e quem controla a inteligencia é o pai. Ver https://react.dev/learn/thinking-in-react#step-4-identify-where-your-state-should-live